lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/LibertyDiagnosticParticipant.java

```diff
@@ -129,7 +129,7 @@ private void validateVariables(DOMDocument domDocument, List<Diagnostic> diagnos
         for (VariableLoc variable : variables) {
             if (!variablesMap.containsKey(variable.getValue())) {
                 String variableInDoc = String.format("${%s}", variable.getValue());
-                Range range = XMLPositionUtility.createRange(variable.getStartLoc(),variable.getEndLoc(),
+                Range range = XMLPositionUtility.createRange(variable.getStartLoc()-2,variable.getEndLoc()+1,
```

Comment:
Add comment to the code on why you are doing `-2` and `+1` here. Give a concrete example.

Reply:
added
//range is used in ReplaceVariable to provide quick fix.
// we just need the variable value range here as ${} is added in replace variable message